python-test/features/steps/local_agent.py

```diff
@@ -37,6 +37,24 @@ def run_local_agent_container(context, port):
     if port not in context.containers_id.keys():
         context.containers_id[str(port)] = context.container_id
 
+@step('the container logs that were output after {condition} contain the message "{text_to_match}" within'
+      '{time_to_wait} seconds')
```

Comment:
Check if we can use this instead, even though it is just a couple of seconds if we want to run things in parallel in the near future, the sleep can block the execution.
https://realpython.com/python-sleep/#using-eventwait

Reply:
@lpegoraro please check if the way i did is the expected way.. If you agree with this way, i'm gonna open a new PR adapting all the tests.. if you have another suggestion or if it is not exactly the way it should be, please let me know